src/map-feature.js

```diff
@@ -246,6 +246,7 @@ export class MapFeature extends HTMLElement {
       this._layer = parentEl._layer;
       _attachedToMap();
     }
+    this._groupEl._feature = this;
```

Comment:
Actually I think I made a mistake about where this should go.  Will fix.

Reply:
Can't fix it short term.  I will work with a different layer construct in the other branch, to avoid `<map-feature>`, as I'm working on `HTMLLayerElement.label` which doesn't require a dependency on `<map-feature>`

Reply:
Actually, this looks like it's going to be an issue, probably going to have to revert this and the associated test until I can figure out how to work around the whole `createmap` issue.

Reply:
ok that makes sense, Yes the `createmap` issue is a source of quite a few issues, It is also interfering with the map-extent development, would be good to treat is separately.

Reply:
@prushforth Should we revert this change? this PR is ready otherwise!

Reply:
Reverted.  Flaky connection today